cloud-nio/cloud-nio-impl-drs/src/main/scala/cloud/nio/impl/drs/DrsConfig.scala

```diff
@@ -0,0 +1,57 @@
+package cloud.nio.impl.drs
+
+import com.typesafe.config.Config
+import net.ceedubs.ficus.Ficus._
+
+import scala.concurrent.duration._
+
+final case class DrsConfig(marthaUrl: String,
+                           numRetries: Int,
+                           waitInitial: FiniteDuration,
+                           waitMaximum: FiniteDuration,
+                           waitMultiplier: Double,
+                          )
+
+object DrsConfig {
+  // If you update these values also update Filesystems.md!
+  private val DefaultNumRetries = 3
```

Comment:
TOL: maybe it would be better to specify these defaults in reference.conf?

Reply:
Hmm... This code is used in multiple places. Outside the engine, `DrsLocalizerMain` currently looks for CLI variables + environment variables and doesn't use conf files. I didn't want to go refactoring it to declare and use a new `reference.conf`, etc.... but I'll think about if there's an easy way to do this.

Reply:
In this case I think that's fine